README.md

```diff
@@ -29,6 +29,20 @@ cd mbed-os-example-crash-reporting
 mbed deploy
 ```
 
+#### Description
+
+This example demonstrates Mbed OS crash-reporting feature and the new APIs associated with it. If you look at the main.cpp, you will see a
+function with signature `void mbed_error_reboot_callback(mbed_error_ctx *error_context)`. This function is the callback called during Mbed-OS 
+initialization if the system rebooted due to a fatal error. This function is called with a pointer to `mbed_error_ctx` structure which captures the
+error context which triggered the reboot. When you run this program(see `main.cpp`), it will force a fatal error by triggering an exception,
+which causes an auto-reboot(warm-reset) of the system. Note that the system captures the error context in a special location in RAM before 
+triggering the auto-reboot. And during the reboot, the system will detect that the reboot was caused by a fatal error and it calls the 
+`mbed_error_reboot_callback()` with the pointer to error context captured in RAM. The callback function records the fact that a reboot
+occurred due to a fatal error(using a local variable named `reboot_error_happened`) and it captures the error context using `mbed_get_reboot_error_info()`
```

Comment:
In general the 2-3 sentences here use the word error a lot it feels like. Not really a great away around that, just a thought as I was reading this.

Reply:
@kegilbert - Thanks for the review, let me push a change to address these comments.